lib/main.dart

```diff
@@ -1,24 +1,60 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
 import 'package:helpwave/pages/landing.dart';
+import 'package:helpwave/services/language_model.dart';
+import 'package:helpwave/services/theme_model.dart';
 import 'package:helpwave/styling/dark_theme.dart';
 import 'package:helpwave/styling/light_theme.dart';
+import 'package:provider/provider.dart';
 
 void main() {
   runApp(const MyApp());
 }
 
+List<Map<String, String>> languages = [
+  {"Name": "English", "Local": "US", "Shortname": "en"},
+  {"Name": "Deutsch", "Local": "DE", "Shortname": "de"}
+];
+
```

Comment:
Unsure whether we want to leave this in the main.dart

Reply:
for the beginning i think it's okay